traffic_control/admin/traffic_light.py

```diff
@@ -24,11 +29,60 @@ class TrafficLightPlanFileInline(admin.TabularInline):
 
 @admin.register(TrafficLightPlan)
 class TrafficLightPlanAdmin(
-    SoftDeleteAdminMixin, UserStampedAdminMixin, admin.OSMGeoAdmin, AuditLogHistoryAdmin
+    EnumChoiceValueDisplayAdminMixin,
+    SoftDeleteAdminMixin,
+    UserStampedAdminMixin,
+    admin.OSMGeoAdmin,
+    AuditLogHistoryAdmin,
 ):
     default_lon = HELSINKI_LONGITUDE
     default_lat = HELSINKI_LATITUDE
     default_zoom = 12
```

Comment:
I've said this before, but these three feel more like settings to me. It's very minor thing over all and does not affect the functionality of course. Just pointing it out here.